sherpa/__init__.py

```diff
@@ -68,7 +69,7 @@ def format(self, record):
 Welcome to Sherpa: CXC's Modeling and Fitting Package
 -----------------------------------------------------
 Version: Sherpa %s Monday, July 19, 2010
```

Comment:
Is this variable actually ever used anywhere? The date - Monday, July 19, 2010 - is rather old and not what I see in the Sherpa banner in CIAO 4.8.